recipe/build.sh

```diff
@@ -15,6 +15,11 @@ unset LDFLAGS
 source activate "${CONDA_DEFAULT_ENV}"
 
 
+# Use OpenBLAS with 1 thread only as it seems to be using too many
+# on the CIs apparently.
+export OPENBLAS_NUM_THREADS=1
```

Comment:
Didn't need it in the build before, but just setting it for good measure. Shouldn't have any effect on how OpenBLAS is used at run-time.